src/Sulu/Component/Content/Metadata/Exception/WrongCacheLifetimeTypeException.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Metadata\Exception;
+
+use Exception;
+
+/**
+ * Wrong cache lifetime type exception.
+ */
+class WrongCacheLifetimeTypeException extends Exception
+{
+    /**
+     * The type causing the error.
+     *
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @param string $type The type causing the error
+     * @param string $message
+     */
+    public function __construct($type, $message = '')
+    {
+        $this->type = $type;
+        parent::__construct($message);
```

Comment:
generate own message like `Type of CacheLifeTime '%s' not supported. Supported Types: "xy", "yz".`